tests/integration/it_reservation_settings_test.py

```diff
@@ -1,44 +1,33 @@
-"""
-@todo #291:30min Enable all tests below when then functionality in for
- ReservationSettings will be enabled. There will be tuning needed because there
- are no endpoint names in the views.
-"""
 import datetime
 from http import HTTPStatus
-
-import pytest
 from flask import url_for
-
 from timeless.reservations import models
 
 
-@pytest.mark.skip
 def test_list(client):
     data = reservation_data("nicer comment here")
-    client.post(url_for("reservations.settings_create"), data=data)
+    client.post(url_for("reservations.settings_create_view"), data=data)
     response = client.get(url_for("reservations.settings_list"))
     assert response.status_code == HTTPStatus.OK
     html = response.data.decode("utf-8")
     assert html.count(data["num_of_persons"]) == 1
     assert html.count(data["comment"]) == 1
 
 
-@pytest.mark.skip
 def test_create(client):
     data = reservation_data("nice comment here")
-    response = client.post(url_for("reservations.settings_create"), data=data)
+    response = client.post(url_for("reservations.settings_create_view"), data=data)
     assert response.status_code == HTTPStatus.FOUND
     assert response.location.endswith(url_for("reservations.settings_list"))
 
 
-@pytest.mark.skip
 def test_edit(client):
     data = reservation_data("nicer comment here")
     new_data = reservation_data("different comment here")
-    client.post(url_for("reservations.settings_create"), data=data)
+    client.post(url_for("reservations.settings_create_view"), data=data)
     identifier = models.ReservationSettings.query.first().id
     client.post(
-        url_for("reservations.settings_create", id=identifier), data=new_data
+        url_for("reservations.settings_create_view", id=identifier), data=new_data
```

Comment:
@VladimirRyzhkov Here there should be the url for settings edit. Do you happen to know what that one is? Maybe ``reservations.settings_edit``? I've no idea :(

Reply:
@amihaiemil  there is no endpoint for updating reservation settings..